src/scrapper/mortgage_endpoints.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, Depends
+
+from library.database.database import async_session_maker
+from library.s3.minio import upload_file_to_s3
+from scrapper.mortgage_model import Mortgage
+from users.user_model import User
+from src.scrapper.mortgage_scrapper import MortgageInputData, get_website_mortgage_result_table
+from src.users.user_manager import current_active_user
+
+mortgage_router = APIRouter(
+)
+
+
+@mortgage_router.post("/")
```

Comment:
please add separate route `/scrap` or `/parse`

Reply:
add separate route "/scrap" instead of "/" for mortgage_router